src/CommonProviderImplementation/QuotationBuilder.fs

```diff
@@ -53,25 +53,18 @@ let (?) (typ:Type) (operation:string) (args1:'T) : 'R =
     | [| :? MethodInfo as mi |] -> 
         let mi = 
           if tyargs = [] then mi
-          else mi.MakeGenericMethod(tyargs |> Array.ofList |> Array.map ProvidedTypeDefinition.EraseType)
```

Comment:
Why is eraseType no longer needed? IIRC it was needed when tyargs contained units of measure